content/userguide/index.md

```diff
@@ -0,0 +1,271 @@
+if you dont have them already.
+
+Open up your terminal and run this script:
+
+```bash
+sudo -E bash -c "$(curl -s https://raw.githubusercontent.com/mysteriumnetwork/node/master/install.sh)"
+```
+
+### Check service health 
+
+After installation you can check the status of the Mysterium node
+
+```bash
+sudo systemctl status mysterium-node.service
+```
+
+If everything is working you should see similar output to this:
+
+```sh
+mysterium@pop-os:~$ sudo systemctl status mysterium-node.service 
+● mysterium-node.service - Server for Mysterium - decentralised VPN Network
+     Loaded: loaded (/lib/systemd/system/mysterium-node.service; enabled; vendor preset: enabled)
+     Active: active (running) since Fri 2020-11-20 00:00:00 GMT;
+       Docs: https://mysterium.network/
+   Main PID: 1920 (myst)
+      Tasks: 30 (limit: 19009)
+     Memory: 73.0M
+```
+
+If thats not the case, you can try and restart the mysterium service by running:
+
+```bash
+sudo systemctl restart mysterium-node.service
+```
+
+If you still seem to not be able to use our 
+
+
+## Basic myst CLI guide 
+
+In order to start using our dVPN you will have to get familiar with our
+`myst cli` console application.
+
+Initial setup shouldn't take more than a few minutes and after that you'll only need to execute
+a few commands whenever you want to use mysterium dVPN.
+
+### How to use the CLI
+
+To start open your terminal and execute:
+```bash
+myst cli
+```
+
+Once started your terminal window should change to the `myst cli` interface.
+To get help and see a list of possible commands execute:
+```bash
+» help 
+```
+
+Each command has an output, if the output does not start with `[ERROR]` consider that command a success.
+
+### Initial setup 
+
+If you've never used mysterium dVPN you'll have to create, unlock and register your identity.
+Go ahead and fire up the cli application and follow the steps below. 
+
+#### Create a new identity 
+> Command: identities new
+
+In order to identify yourself in the network you must have an identity, to create it
+execute the following command in the `cli`:
+
+```bash
+» identities new
+```
+
+This command will create you a new identity which you can use.
+It will be printed to your terminal, but in case you want to check it out later you can do that
+will the following command:
+
+```bash
+» identities list 
+```
+
+#### Unlock the new identity 
+> Command: identities unclock [identity] 
+
+In order to use the created identity it has to be unclocked first.
+
+**Unlocking has to be done every time we launch the `myst cli` and want to use the created identity.**
+
+For the sake of simplicity let's say we've received an identity: `0x4570fe47a49af9ae9bd76f029818413ea18620000`,
+which we'll use in all later steps.
+
+To unlock it we'd have to execute:
+```bash
+» identities unlock 0x4570fe47a49af9ae9bd76f029818413ea18620000
+```
```

Comment:
You might want to add `tab` feature. If you type in `identities unlock ` and keep hitting tab, you will start cycling through all identities.

Reply:
Also tab works with other commands

Reply:
Added a general explanation on this in the `How to use the CLI` section